frontend/tests/controllers/ContestCloneTest.php

```diff
@@ -180,4 +180,35 @@ public function testCreatePrivateContestCloneWithoutAccess() {
             $this->assertSame('userNotAllowed', $e->getMessage());
         }
     }
+
+    /*
+     * Under13 users can't clone contests.
+     */
+    public function testUserUnder13CannotCloneContests() {
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest();
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams([
+                'birth_date' => strtotime('-10 years'),
+            ]),
+        );
```

Comment:
In order to ensure the test will not fail in a future, we can set a default date for the test:
```suggestion
        $defaultDate = strtotime('10 September 2022');
        \OmegaUp\Time::setTimeForTesting($defaultDate);
        // Create a 10 years-old user
        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
            new \OmegaUp\Test\Factories\UserParams([
                'birthDate' => strtotime('20 September 2012'),
            ]),
        );
```
otherwise, we cannot guarantee user is 10 years old.

Reply:
if possible use `2022-09-10T00:00:00Z` instead of `10 September 2022` in all the changes.